src/components/interpolatedColorLegend.ts

```diff
@@ -3,19 +3,19 @@
 module Plottable {
 export module Components {
   export class InterpolatedColorLegend extends Component {
-    private _measurer: SVGTypewriter.Measurers.Measurer;
-    private _wrapper: SVGTypewriter.Wrappers.Wrapper;
-    private _writer: SVGTypewriter.Writers.Writer;
-    private _scale: Scales.InterpolatedColor;
+    private measurer: SVGTypewriter.Measurers.Measurer;
+    private wrapper: SVGTypewriter.Wrappers.Wrapper;
+    private writer: SVGTypewriter.Writers.Writer;
+    private scale: Scales.InterpolatedColor;
     private _orientation: String ;
```

Comment:
Can we make without underscore or save it for later if we rename the corresponding function to orientation?